odoo/exceptions.py

```diff
@@ -13,24 +13,27 @@
 import logging
 from inspect import currentframe
 from .tools.func import frame_codeinfo
+from .tools.translate import _
 
 _logger = logging.getLogger(__name__)
 
 
 # kept for backward compatibility
 class except_orm(Exception):
-    def __init__(self, name, value=None):
+    def __init__(self, name, value=None, actions=None):
         if type(self) == except_orm:
             caller = frame_codeinfo(currentframe(), 1)
-            _logger.warn('except_orm is deprecated. Please use specific exceptions like UserError or AccessError. Caller: %s:%s', *caller)
+            _logger.warn('except_orm is deprecated. Please use specific exceptions like UserError '
+                         'or AccessError. Caller: %s:%s', *caller)
         self.name = name
         self.value = value
         self.args = (name, value)
+        self.actions = actions or []
```

Comment:
remove from signature change, juste add actions = False to document the feature. Dont reindent other lines.

Reply:
why not keep it in master and then make a backportable PR?